apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c

```diff
@@ -114,13 +115,13 @@ int wifiAutoConnectInit()
 	return -1;
 }
 
-void wifiAutoConnectDeInit()
+static void wifiAutoConnectDeInit(void)
 {
 	isConnected = 0;
 	dm_conn_unregister_linkup_cb(linkUpEvent);
 }
 
-static void wifiAutoConnect()
+static int wifiAutoConnect(void)
 {
 	int ret;
 	if ((ret = wifiAutoConnectInit()) == 1) {
```

Comment:
Maybe we don't need ret in the first call itself. 
if (wifiAutoConnectInit() == 1) ...